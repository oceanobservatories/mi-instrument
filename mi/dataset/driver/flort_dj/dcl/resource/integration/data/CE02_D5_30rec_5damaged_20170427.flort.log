2017/04/27 00:00:13.777 [flort:DLOGP2]:Instrument sampling - Data logger entering data acquisition state
2017/04/27 11:45:45.249 04/27/17	11:44:33	700	225	695	169	460	63	556
2017/04/27 11:45:46.379 04/27/17	11:44:34	700	237	695	150	460	61	556
2017/04/27 11:45:47.509 04/27/17	11:44:35	700	219	695	143	460	65	556
2017/04/27 11:45:54.286 04/27/17	11:44:42	700	225	695	149	460	63	556
2017/04/27 11:46:21.394 04/27/17	11:45:09	700	216	695	-163	460	63	555
2017/04/27 11:46:22.524 04/27/17	11:45:10	700	232	695	141	460	62	555
2017/04/27 11:46:23.653 04/27/17	11:45:11	700	229	695	143	460	62	555
2017/04/27 11:46:24.789 04/27/17	11:45:12	700	225	695	159	460	66	555
2017/04/27 11:46:51.893 04/27/17	11:45:40	700	225	695	161	460	64	554
2017/04/27 11:46:53.022 04/27/17	11:45:41	700	212	695	158	460	63	554
2017/04/27 19:18:01.339 [flort:DLOGP2]:Instrument Stopped [Power Off]
2017/04/27 19:30:14.409 [flort:DLOGP2]:Instrument sampling - Data logger entering data acquisition state
2017/04/27 19:30:15.576 04/27/17	19:29:03	700	263	695	166	460	63	557
2017/04/27 19:30:16.705 04/27/17	19:29:04	700	272	695	177	460	62	557
2017/04/27 19:30:27.835 04/27/17	19:29:05	700	267	695	179	460	62	557
2017/04/27 19:30:28.964 04/27/17	19:29:06	700	271	695	184	460	65	557
2017/04/27 19:33:00.722 [flort:DLOGP2]:Instrument Stopped [Power Off]
2017/04/27 19:45:12.894 [flort:DLOGP2]:Instrument sampling - Data logger entering data acquisition state
2017/04/27 23:32:17.933 04/27/17	23:31:05	700	280	695	166	460	61	554
2017/04/27 23:32:19.061 04/27/17	23:31:06	700	282	695	173	460	63	553
2017/04/27 23:32:20.190 04/27/17	23.31.08	700	281	695	190	460	61	554
2017/04/27 23:32:21.319 04/27/17	23:31:09	700	305	695	166	460	61	553
2017/04/27 23:32:22.449 04/27/17	23:31:10	700	286	695	174	460	62	553
2017/04/27 23:32:52.952 04/27/17	23:31:40	700	276	695	176	460	62	553
2017/04/27 23:33:02.198 [flort:DLOGP2]:Instrument Stopped [Power Off]
2017/04/27 23:45:14.302 [flort:DLOGP2]:Instrument sampling - Data logger entering data acquisition state
2017/04/27 23:46:20.991 04/27/17	23:45:08	700	272	695	194	460	62	555
2017/04/27 23:46:22.121 04/27/17	23:45:10	700	324	695	178	460	64	555
2017/04/27 23:46:43.586 04/27/17	23:45:31	-700	267	695	176	460	61	554
2017/04/27 23:46:44.716 04/27/17	23:45:32	700	291	695	169	460	62	554
2017/04/27 23:46:45.846 04/27/17	23:45:33	700	294	695	178	460	61	554
2017/04/27 23:46:46.975 04/27/17	23:45:34	700	283	695	198	460	63	554
2017/04/27 23:47:17.478 04/27/17	23:46:05	700	279	695	166	460	63	554
2017/04/27 23:47:18.607 04/27/17	23:46:06	700	282	695	164	460	62	554
2017/04/27 23:47:19.737 04/27/17	23:46:07	700	282	695	158	460	62	553
2017/04/27 23:47:26.515 04/27/17	23:46:14	700	276	695	171	460	59.7	553
2017/04/27 23:47:27.644 04/27/17	23:46:15	700	281	695	165	460	62	553
2017/04/27 23:47:28.775 04/27/17	23:46:16	700	293	695	177	460	66	553
2017/04/27 23:47:59.277 04/27/17	23:46:47	700	271	695	151	460	60	553
2017/04/27 23:48:00.407 04/27/17	23:46:48	700	288	695	171	460	63	.553
2017/04/27 23:48:01.537 04/27/17	23:46:49	700	281	695	162	460	60	553
2017/04/27 23:48:01.745 [flort:DLOGP2]:Instrument Stopped [Power Off]
