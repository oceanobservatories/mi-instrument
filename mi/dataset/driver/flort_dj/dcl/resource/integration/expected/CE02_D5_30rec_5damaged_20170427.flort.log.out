
 subsite  | node  | sensor       | bin        | method      | time                  | deployment | internal_timestamp | port_timestamp        | preferred_timestamp | quality_flag | date_string | dcl_controller_timestamp | measurement_wavelength_beta | measurement_wavelength_cdom | measurement_wavelength_chl | raw_internal_temp | raw_signal_beta | raw_signal_cdom | raw_signal_chl | time_string
----------+-------+--------------+------------+-------------+-----------------------+------------+--------------------+-----------------------+---------------------+--------------+-------------+--------------------------+-----------------------------+-----------------------------+----------------------------+-------------------+-----------------+-----------------+----------------+-------------
 CE02SHSM | RID27 | 02-FLORTD000 | 3702240000 | telemetered | 3702282345.2490000725 |          5 |         3702282273 | 3702282345.2490000725 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               556 |             225 |              63 |            169 |        null
 CE02SHSM | RID27 | 02-FLORTD000 | 3702240000 | telemetered | 3702282346.3789997101 |          5 |         3702282274 | 3702282346.3789997101 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               556 |             237 |              61 |            150 |        null
 CE02SHSM | RID27 | 02-FLORTD000 | 3702240000 | telemetered | 3702282347.5089998245 |          5 |         3702282275 | 3702282347.5089998245 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               556 |             219 |              65 |            143 |        null
 CE02SHSM | RID27 | 02-FLORTD000 | 3702240000 | telemetered | 3702282354.2860002518 |          5 |         3702282282 | 3702282354.2860002518 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               556 |             225 |              63 |            149 |        null
 CE02SHSM | RID27 | 02-FLORTD000 | 3702240000 | telemetered | 3702282382.5240001678 |          5 |         3702282310 | 3702282382.5240001678 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               555 |             232 |              62 |            141 |        null
 CE02SHSM | RID27 | 02-FLORTD000 | 3702240000 | telemetered | 3702282383.6529998779 |          5 |         3702282311 | 3702282383.6529998779 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               555 |             229 |              62 |            143 |        null
 CE02SHSM | RID27 | 02-FLORTD000 | 3702240000 | telemetered | 3702282384.7890000343 |          5 |         3702282312 | 3702282384.7890000343 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               555 |             225 |              66 |            159 |        null
 CE02SHSM | RID27 | 02-FLORTD000 | 3702240000 | telemetered |  3702282411.892999649 |          5 |         3702282340 |  3702282411.892999649 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               554 |             225 |              64 |            161 |        null
 CE02SHSM | RID27 | 02-FLORTD000 | 3702240000 | telemetered | 3702282413.0220003128 |          5 |         3702282341 | 3702282413.0220003128 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               554 |             212 |              63 |            158 |        null
 CE02SHSM | RID27 | 02-FLORTD000 | 3702240000 | telemetered | 3702310215.5760002136 |          5 |         3702310143 | 3702310215.5760002136 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               557 |             263 |              63 |            166 |        null
 CE02SHSM | RID27 | 02-FLORTD000 | 3702240000 | telemetered | 3702310216.7049999237 |          5 |         3702310144 | 3702310216.7049999237 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               557 |             272 |              62 |            177 |        null
 CE02SHSM | RID27 | 02-FLORTD000 | 3702240000 | telemetered | 3702310227.8350000381 |          5 |         3702310145 | 3702310227.8350000381 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               557 |             267 |              62 |            179 |        null
 CE02SHSM | RID27 | 02-FLORTD000 | 3702240000 | telemetered | 3702310228.9639997482 |          5 |         3702310146 | 3702310228.9639997482 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               557 |             271 |              65 |            184 |        null
 CE02SHSM | RID27 | 02-FLORTD000 | 3702240000 | telemetered | 3702324737.9330000877 |          5 |         3702324665 | 3702324737.9330000877 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               554 |             280 |              61 |            166 |        null
 CE02SHSM | RID27 | 02-FLORTD000 | 3702240000 | telemetered | 3702324739.0609998703 |          5 |         3702324666 | 3702324739.0609998703 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               553 |             282 |              63 |            173 |        null
 CE02SHSM | RID27 | 02-FLORTD000 | 3702240000 | telemetered | 3702324741.3190002441 |          5 |         3702324669 | 3702324741.3190002441 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               553 |             305 |              61 |            166 |        null
 CE02SHSM | RID27 | 02-FLORTD000 | 3702240000 | telemetered | 3702324742.4489998817 |          5 |         3702324670 | 3702324742.4489998817 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               553 |             286 |              62 |            174 |        null
 CE02SHSM | RID27 | 02-FLORTD000 | 3702240000 | telemetered | 3702324772.9519996643 |          5 |         3702324700 | 3702324772.9519996643 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               553 |             276 |              62 |            176 |        null
 CE02SHSM | RID27 | 02-FLORTD000 | 3702240000 | telemetered | 3702325580.9910001755 |          5 |         3702325508 | 3702325580.9910001755 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               555 |             272 |              62 |            194 |        null
 CE02SHSM | RID27 | 02-FLORTD000 | 3702240000 | telemetered | 3702325582.1210002899 |          5 |         3702325510 | 3702325582.1210002899 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               555 |             324 |              64 |            178 |        null
 CE02SHSM | RID27 | 02-FLORTD000 | 3702240000 | telemetered | 3702325604.7160000801 |          5 |         3702325532 | 3702325604.7160000801 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               554 |             291 |              62 |            169 |        null
 CE02SHSM | RID27 | 02-FLORTD000 | 3702240000 | telemetered | 3702325605.8459997177 |          5 |         3702325533 | 3702325605.8459997177 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               554 |             294 |              61 |            178 |        null
 CE02SHSM | RID27 | 02-FLORTD000 | 3702240000 | telemetered | 3702325606.9749999046 |          5 |         3702325534 | 3702325606.9749999046 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               554 |             283 |              63 |            198 |        null
 CE02SHSM | RID27 | 02-FLORTD000 | 3702240000 | telemetered | 3702325637.4779996872 |          5 |         3702325565 | 3702325637.4779996872 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               554 |             279 |              63 |            166 |        null
 CE02SHSM | RID27 | 02-FLORTD000 | 3702240000 | telemetered |  3702325638.607000351 |          5 |         3702325566 |  3702325638.607000351 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               554 |             282 |              62 |            164 |        null
 CE02SHSM | RID27 | 02-FLORTD000 | 3702240000 | telemetered | 3702325639.7369999886 |          5 |         3702325567 | 3702325639.7369999886 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               553 |             282 |              62 |            158 |        null
 CE02SHSM | RID27 | 02-FLORTD000 | 3702240000 | telemetered | 3702325647.6440000534 |          5 |         3702325575 | 3702325647.6440000534 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               553 |             281 |              62 |            165 |        null
 CE02SHSM | RID27 | 02-FLORTD000 | 3702240000 | telemetered | 3702325648.7750000954 |          5 |         3702325576 | 3702325648.7750000954 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               553 |             293 |              66 |            177 |        null
 CE02SHSM | RID27 | 02-FLORTD000 | 3702240000 | telemetered | 3702325679.2769999504 |          5 |         3702325607 | 3702325679.2769999504 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               553 |             271 |              60 |            151 |        null
 CE02SHSM | RID27 | 02-FLORTD000 | 3702240000 | telemetered | 3702325681.5369997025 |          5 |         3702325609 | 3702325681.5369997025 |      port_timestamp |           ok |        null |                     null |                         700 |                         460 |                        695 |               553 |             281 |              60 |            162 |        null

(30 rows)
